tests/mints5/input.dat

```diff
@@ -311,6 +311,7 @@ compare_strings("D3", ethane_gauche.get_full_point_group(), "D3 point group");
 
 
 molecule triplet_ethylene {
+  0 3
```

Comment:
Is there a new output reference file for this test?

Reply:
You'd think so, but this test is solely concerned with geometry and orientation and doesn't actually use the multiplicity. I have a test in another repo that does store HF energies for all these, so I noticed the multiplicity was missing.